generate.go

```diff
@@ -1,4 +1,38 @@
-//go:generate go-bindata -pkg=data -ignore=.*\.go -o data.go . ./benkeen/d3pie/ ./jiahuang/d3-timeline/ ./krisk/fuse/ ./zeroclipboard/zeroclipboard/ ./seiyria/bootstrap-slider/
-//go:generate go fmt .
+// +build generate
 
-package data
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+	pathpkg "path"
```

Comment:
Fair enough. I normally also avoid renaming packages, but I've seen that `pathpkg "path"` is a pretty common occurrence in the Go standard (and external) libraries:

- http://gotools.org/go/build#build-go-L19
- http://gotools.org/cmd/go#pkg-go-L18
- http://gotools.org/golang.org/x/tools/godoc/vfs#namespace-go-L11
- http://gotools.org/golang.org/x/tools/godoc/vfs#os-go-L11
- http://gotools.org/golang.org/x/tools/godoc (8 occurrences)

In fact, it's even in the example provided for [`astutil.AddNamedImport`](https://godoc.org/golang.org/x/tools/go/ast/astutil#AddNamedImport).

That's why I adopted this pattern of renaming "path" to `pathpkg` specifically.

I will change it to avoid renaming the package if you prefer that, it's not a problem.